modules/header/react/lib/parts/Search.tsx

```diff
@@ -274,19 +293,27 @@ export class Search extends React.Component<SearchProps, SearchState> {
           mountOnEnter={true}
           unmountOnExit={true}
         >
-          <SearchContainer onSubmit={this.onSearchSubmit} {...props}>
+          <SearchContainer
+            rightAlign={rightAlign}
+            collapse={collapse}
+            onSubmit={this.onSearchSubmit}
+            {...elemProps}
+          >
             <SearchInput
-              {...props}
-              type="search"
+              placeholder={placeholder}
+              themeColor={themeColor}
+              collapse={collapse}
+              role="search"
```

Comment:
Do you mean to put `role="search"` on the `<SearchContainer>` instead? I couldn't find any documentation for `role="search"` on inputs. https://www.w3.org/TR/wai-aria-practices/examples/landmarks/search.html

Reply:
whoops you are correct, will fix

Reply:
updated! 😄 nice catch